_posts/2016-06-22-hello-beercraft.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Maku brewing and Solita's passion pale ale
+author: Rinorragi
+excerpt: Solita ordered passion pale ale from Maku brewing and got introduction to their craftmenship
+tags: 
+- Beercraft
+---
+Solita wanted to create a beer of their own. The process was like a software project. First we tried to figure out what we want. We had our internal voting for the taste and name for the beer and "Intohimo" passion pale ale won the contest. After knowing what we need we went and sent multiple offers to different breweries. Among the offers we selected Maku brewing and decided with them that the receipt needs to be tested first. Maku brewing created 20l patch of the receipt they invented for us and at 21.6.2016 we got an opportunity to taste the beer and give feedback until final brew. All started with a bus trip from Solita's office at Helsinki to Tuusula. We had a great time already in the bus in discussion of projects and expectations of the upcoming beer. 
+
+## Maku brewing
+
+[Maku brewing](http://www.makubrewing.com/) is few years old finnish craft beer brewery. They are making great beers. Their IPA is most likely the best finnish IPA and it got some awards as well. Their amber ale is also great and it has been awarder as well. 
```

Comment:
Ja yhdistäisin noi kaksi lausetta näin:
[Maku brewing](http://www.makubrewing.com/) is a couple years old finnish craft beer brewery making great beers.